docs/docarray-v2.md

```diff
@@ -115,12 +113,27 @@ with Deployment(uses=MyExec) as dep:
 ```
 
 
+## Compatible Features
+
+Jina is working to offer full compatibility of the Jina framework with the power of the new DocArray version.
+
+At this moment, these are the features supported if the APIs described in the previous sections are used.
+
+- All the features offered by {ref}`Deployment <deployment>` where a single Executor is served and {ref}`scaled <scale-out>`. This support includes the usage of both
+HTTP and gRPC protocols and the usage of both of them at the same type.
+
+- When combining multiple Deployments in a pipeline using a {ref}`Flow <flow-cookbook>`, multiple limitations exist at this moment:
+
+    - Only gRPC protocol is supported.
+    - Only linear Flows are supported, no topologies using bifurcations can be used at the moment.
```

Comment:
why is that ?

Reply:
reduce method is not added for now in the Flow, this will be worked on later

Reply:
makes sense